CalculatorTest.py

```diff
@@ -20,6 +20,9 @@ def test_run_operation_divide(self):
         with self.assertRaises(ZeroDivisionError):
             Calculator.run_operation(9, 0, '/')
 
+    def test_run_operation_modulo(self):
+        self.assertEqual(7, Calculator.run_operation(7, 10, '%'))
+
```

Comment:
I'd love to see more test cases for the modulo operation! For example, what should happen if the user inputs a float as one of the arguments? a negative number?

Reply:
I agree! Adding the modulo is such a great idea and should be tested as @joshfeli said!

Reply:
I do, however, love the structure of this test case: it matches that of the other test cases and is overall nicely written!

Reply:
To further expand the test cases, there should also be a modulo by 0 case, which raises an error in Python.

Reply:
I agree with Panagris' suggestion. Not only does it ensure that we are correctly handling modulo by 0, but it also matches the format of the `test_run_operation_divide` method.